server/src/index.ts

```diff
@@ -1,9 +1,20 @@
+import { PrismaClient } from "@prisma/client";
 import { GraphQLServer } from "graphql-yoga";
 import { resolvers } from "./resolvers";
 
+const prisma = new PrismaClient();
+
+export type prismaClientType = typeof prisma;
+
 const server = new GraphQLServer({
   typeDefs: "./src/schema.graphql",
   resolvers,
+  context(request) {
+    return {
```

Comment:
typescript doesn't support inline returns?

And if you want, you can add an api check here. For example, add an X API KEY. Use bcrypt to decrypt and check and if the secret API key matches, then only allow the request to go ahead, else throw an unauth error.